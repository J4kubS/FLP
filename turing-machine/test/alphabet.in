S   F R
S a A a
S b B b
S c C c
S d D d
S e E e
S f G g
S g G g
S h H h
S i I i
S j J j
S k K k
S l L l
S m M m
S n N n
S o O o
S p P p
S q Q q
S r R r
S s T t
S t T t
S u U u
S v V v
S w W w
S x X x
S y Y y
S z Z z
A a B b
B b C c
C c D d
D d E e
E e G g
G g H h
H h I i
I i J j
J j K k
K k L l
L l M m
M m N n
N n O o
O o P p
P p Q q
Q q R r
R r T t
T t U u
U u V v
V v W w
W w X x
X x Y y
Y y Z z
Z z S R
abcdefghijklmnopqrstuvwxyz
