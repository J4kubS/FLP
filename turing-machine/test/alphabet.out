Sabcdefghijklmnopqrstuvwxyz
Aabcdefghijklmnopqrstuvwxyz
Bbbcdefghijklmnopqrstuvwxyz
Ccbcdefghijklmnopqrstuvwxyz
Ddbcdefghijklmnopqrstuvwxyz
Eebcdefghijklmnopqrstuvwxyz
Ggbcdefghijklmnopqrstuvwxyz
Hhbcdefghijklmnopqrstuvwxyz
Iibcdefghijklmnopqrstuvwxyz
Jjbcdefghijklmnopqrstuvwxyz
Kkbcdefghijklmnopqrstuvwxyz
Llbcdefghijklmnopqrstuvwxyz
Mmbcdefghijklmnopqrstuvwxyz
Nnbcdefghijklmnopqrstuvwxyz
Oobcdefghijklmnopqrstuvwxyz
Ppbcdefghijklmnopqrstuvwxyz
Qqbcdefghijklmnopqrstuvwxyz
Rrbcdefghijklmnopqrstuvwxyz
Ttbcdefghijklmnopqrstuvwxyz
Uubcdefghijklmnopqrstuvwxyz
Vvbcdefghijklmnopqrstuvwxyz
Wwbcdefghijklmnopqrstuvwxyz
Xxbcdefghijklmnopqrstuvwxyz
Yybcdefghijklmnopqrstuvwxyz
Zzbcdefghijklmnopqrstuvwxyz
zSbcdefghijklmnopqrstuvwxyz
zBbcdefghijklmnopqrstuvwxyz
zCccdefghijklmnopqrstuvwxyz
zDdcdefghijklmnopqrstuvwxyz
zEecdefghijklmnopqrstuvwxyz
zGgcdefghijklmnopqrstuvwxyz
zHhcdefghijklmnopqrstuvwxyz
zIicdefghijklmnopqrstuvwxyz
zJjcdefghijklmnopqrstuvwxyz
zKkcdefghijklmnopqrstuvwxyz
zLlcdefghijklmnopqrstuvwxyz
zMmcdefghijklmnopqrstuvwxyz
zNncdefghijklmnopqrstuvwxyz
zOocdefghijklmnopqrstuvwxyz
zPpcdefghijklmnopqrstuvwxyz
zQqcdefghijklmnopqrstuvwxyz
zRrcdefghijklmnopqrstuvwxyz
zTtcdefghijklmnopqrstuvwxyz
zUucdefghijklmnopqrstuvwxyz
zVvcdefghijklmnopqrstuvwxyz
zWwcdefghijklmnopqrstuvwxyz
zXxcdefghijklmnopqrstuvwxyz
zYycdefghijklmnopqrstuvwxyz
zZzcdefghijklmnopqrstuvwxyz
zzScdefghijklmnopqrstuvwxyz
zzCcdefghijklmnopqrstuvwxyz
zzDddefghijklmnopqrstuvwxyz
zzEedefghijklmnopqrstuvwxyz
zzGgdefghijklmnopqrstuvwxyz
zzHhdefghijklmnopqrstuvwxyz
zzIidefghijklmnopqrstuvwxyz
zzJjdefghijklmnopqrstuvwxyz
zzKkdefghijklmnopqrstuvwxyz
zzLldefghijklmnopqrstuvwxyz
zzMmdefghijklmnopqrstuvwxyz
zzNndefghijklmnopqrstuvwxyz
zzOodefghijklmnopqrstuvwxyz
zzPpdefghijklmnopqrstuvwxyz
zzQqdefghijklmnopqrstuvwxyz
zzRrdefghijklmnopqrstuvwxyz
zzTtdefghijklmnopqrstuvwxyz
zzUudefghijklmnopqrstuvwxyz
zzVvdefghijklmnopqrstuvwxyz
zzWwdefghijklmnopqrstuvwxyz
zzXxdefghijklmnopqrstuvwxyz
zzYydefghijklmnopqrstuvwxyz
zzZzdefghijklmnopqrstuvwxyz
zzzSdefghijklmnopqrstuvwxyz
zzzDdefghijklmnopqrstuvwxyz
zzzEeefghijklmnopqrstuvwxyz
zzzGgefghijklmnopqrstuvwxyz
zzzHhefghijklmnopqrstuvwxyz
zzzIiefghijklmnopqrstuvwxyz
zzzJjefghijklmnopqrstuvwxyz
zzzKkefghijklmnopqrstuvwxyz
zzzLlefghijklmnopqrstuvwxyz
zzzMmefghijklmnopqrstuvwxyz
zzzNnefghijklmnopqrstuvwxyz
zzzOoefghijklmnopqrstuvwxyz
zzzPpefghijklmnopqrstuvwxyz
zzzQqefghijklmnopqrstuvwxyz
zzzRrefghijklmnopqrstuvwxyz
zzzTtefghijklmnopqrstuvwxyz
zzzUuefghijklmnopqrstuvwxyz
zzzVvefghijklmnopqrstuvwxyz
zzzWwefghijklmnopqrstuvwxyz
zzzXxefghijklmnopqrstuvwxyz
zzzYyefghijklmnopqrstuvwxyz
zzzZzefghijklmnopqrstuvwxyz
zzzzSefghijklmnopqrstuvwxyz
zzzzEefghijklmnopqrstuvwxyz
zzzzGgfghijklmnopqrstuvwxyz
zzzzHhfghijklmnopqrstuvwxyz
zzzzIifghijklmnopqrstuvwxyz
zzzzJjfghijklmnopqrstuvwxyz
zzzzKkfghijklmnopqrstuvwxyz
zzzzLlfghijklmnopqrstuvwxyz
zzzzMmfghijklmnopqrstuvwxyz
zzzzNnfghijklmnopqrstuvwxyz
zzzzOofghijklmnopqrstuvwxyz
zzzzPpfghijklmnopqrstuvwxyz
zzzzQqfghijklmnopqrstuvwxyz
zzzzRrfghijklmnopqrstuvwxyz
zzzzTtfghijklmnopqrstuvwxyz
zzzzUufghijklmnopqrstuvwxyz
zzzzVvfghijklmnopqrstuvwxyz
zzzzWwfghijklmnopqrstuvwxyz
zzzzXxfghijklmnopqrstuvwxyz
zzzzYyfghijklmnopqrstuvwxyz
zzzzZzfghijklmnopqrstuvwxyz
zzzzzSfghijklmnopqrstuvwxyz
zzzzzGgghijklmnopqrstuvwxyz
zzzzzHhghijklmnopqrstuvwxyz
zzzzzIighijklmnopqrstuvwxyz
zzzzzJjghijklmnopqrstuvwxyz
zzzzzKkghijklmnopqrstuvwxyz
zzzzzLlghijklmnopqrstuvwxyz
zzzzzMmghijklmnopqrstuvwxyz
zzzzzNnghijklmnopqrstuvwxyz
zzzzzOoghijklmnopqrstuvwxyz
zzzzzPpghijklmnopqrstuvwxyz
zzzzzQqghijklmnopqrstuvwxyz
zzzzzRrghijklmnopqrstuvwxyz
zzzzzTtghijklmnopqrstuvwxyz
zzzzzUughijklmnopqrstuvwxyz
zzzzzVvghijklmnopqrstuvwxyz
zzzzzWwghijklmnopqrstuvwxyz
zzzzzXxghijklmnopqrstuvwxyz
zzzzzYyghijklmnopqrstuvwxyz
zzzzzZzghijklmnopqrstuvwxyz
zzzzzzSghijklmnopqrstuvwxyz
zzzzzzGghijklmnopqrstuvwxyz
zzzzzzHhhijklmnopqrstuvwxyz
zzzzzzIihijklmnopqrstuvwxyz
zzzzzzJjhijklmnopqrstuvwxyz
zzzzzzKkhijklmnopqrstuvwxyz
zzzzzzLlhijklmnopqrstuvwxyz
zzzzzzMmhijklmnopqrstuvwxyz
zzzzzzNnhijklmnopqrstuvwxyz
zzzzzzOohijklmnopqrstuvwxyz
zzzzzzPphijklmnopqrstuvwxyz
zzzzzzQqhijklmnopqrstuvwxyz
zzzzzzRrhijklmnopqrstuvwxyz
zzzzzzTthijklmnopqrstuvwxyz
zzzzzzUuhijklmnopqrstuvwxyz
zzzzzzVvhijklmnopqrstuvwxyz
zzzzzzWwhijklmnopqrstuvwxyz
zzzzzzXxhijklmnopqrstuvwxyz
zzzzzzYyhijklmnopqrstuvwxyz
zzzzzzZzhijklmnopqrstuvwxyz
zzzzzzzShijklmnopqrstuvwxyz
zzzzzzzHhijklmnopqrstuvwxyz
zzzzzzzIiijklmnopqrstuvwxyz
zzzzzzzJjijklmnopqrstuvwxyz
zzzzzzzKkijklmnopqrstuvwxyz
zzzzzzzLlijklmnopqrstuvwxyz
zzzzzzzMmijklmnopqrstuvwxyz
zzzzzzzNnijklmnopqrstuvwxyz
zzzzzzzOoijklmnopqrstuvwxyz
zzzzzzzPpijklmnopqrstuvwxyz
zzzzzzzQqijklmnopqrstuvwxyz
zzzzzzzRrijklmnopqrstuvwxyz
zzzzzzzTtijklmnopqrstuvwxyz
zzzzzzzUuijklmnopqrstuvwxyz
zzzzzzzVvijklmnopqrstuvwxyz
zzzzzzzWwijklmnopqrstuvwxyz
zzzzzzzXxijklmnopqrstuvwxyz
zzzzzzzYyijklmnopqrstuvwxyz
zzzzzzzZzijklmnopqrstuvwxyz
zzzzzzzzSijklmnopqrstuvwxyz
zzzzzzzzIijklmnopqrstuvwxyz
zzzzzzzzJjjklmnopqrstuvwxyz
zzzzzzzzKkjklmnopqrstuvwxyz
zzzzzzzzLljklmnopqrstuvwxyz
zzzzzzzzMmjklmnopqrstuvwxyz
zzzzzzzzNnjklmnopqrstuvwxyz
zzzzzzzzOojklmnopqrstuvwxyz
zzzzzzzzPpjklmnopqrstuvwxyz
zzzzzzzzQqjklmnopqrstuvwxyz
zzzzzzzzRrjklmnopqrstuvwxyz
zzzzzzzzTtjklmnopqrstuvwxyz
zzzzzzzzUujklmnopqrstuvwxyz
zzzzzzzzVvjklmnopqrstuvwxyz
zzzzzzzzWwjklmnopqrstuvwxyz
zzzzzzzzXxjklmnopqrstuvwxyz
zzzzzzzzYyjklmnopqrstuvwxyz
zzzzzzzzZzjklmnopqrstuvwxyz
zzzzzzzzzSjklmnopqrstuvwxyz
zzzzzzzzzJjklmnopqrstuvwxyz
zzzzzzzzzKkklmnopqrstuvwxyz
zzzzzzzzzLlklmnopqrstuvwxyz
zzzzzzzzzMmklmnopqrstuvwxyz
zzzzzzzzzNnklmnopqrstuvwxyz
zzzzzzzzzOoklmnopqrstuvwxyz
zzzzzzzzzPpklmnopqrstuvwxyz
zzzzzzzzzQqklmnopqrstuvwxyz
zzzzzzzzzRrklmnopqrstuvwxyz
zzzzzzzzzTtklmnopqrstuvwxyz
zzzzzzzzzUuklmnopqrstuvwxyz
zzzzzzzzzVvklmnopqrstuvwxyz
zzzzzzzzzWwklmnopqrstuvwxyz
zzzzzzzzzXxklmnopqrstuvwxyz
zzzzzzzzzYyklmnopqrstuvwxyz
zzzzzzzzzZzklmnopqrstuvwxyz
zzzzzzzzzzSklmnopqrstuvwxyz
zzzzzzzzzzKklmnopqrstuvwxyz
zzzzzzzzzzLllmnopqrstuvwxyz
zzzzzzzzzzMmlmnopqrstuvwxyz
zzzzzzzzzzNnlmnopqrstuvwxyz
zzzzzzzzzzOolmnopqrstuvwxyz
zzzzzzzzzzPplmnopqrstuvwxyz
zzzzzzzzzzQqlmnopqrstuvwxyz
zzzzzzzzzzRrlmnopqrstuvwxyz
zzzzzzzzzzTtlmnopqrstuvwxyz
zzzzzzzzzzUulmnopqrstuvwxyz
zzzzzzzzzzVvlmnopqrstuvwxyz
zzzzzzzzzzWwlmnopqrstuvwxyz
zzzzzzzzzzXxlmnopqrstuvwxyz
zzzzzzzzzzYylmnopqrstuvwxyz
zzzzzzzzzzZzlmnopqrstuvwxyz
zzzzzzzzzzzSlmnopqrstuvwxyz
zzzzzzzzzzzLlmnopqrstuvwxyz
zzzzzzzzzzzMmmnopqrstuvwxyz
zzzzzzzzzzzNnmnopqrstuvwxyz
zzzzzzzzzzzOomnopqrstuvwxyz
zzzzzzzzzzzPpmnopqrstuvwxyz
zzzzzzzzzzzQqmnopqrstuvwxyz
zzzzzzzzzzzRrmnopqrstuvwxyz
zzzzzzzzzzzTtmnopqrstuvwxyz
zzzzzzzzzzzUumnopqrstuvwxyz
zzzzzzzzzzzVvmnopqrstuvwxyz
zzzzzzzzzzzWwmnopqrstuvwxyz
zzzzzzzzzzzXxmnopqrstuvwxyz
zzzzzzzzzzzYymnopqrstuvwxyz
zzzzzzzzzzzZzmnopqrstuvwxyz
zzzzzzzzzzzzSmnopqrstuvwxyz
zzzzzzzzzzzzMmnopqrstuvwxyz
zzzzzzzzzzzzNnnopqrstuvwxyz
zzzzzzzzzzzzOonopqrstuvwxyz
zzzzzzzzzzzzPpnopqrstuvwxyz
zzzzzzzzzzzzQqnopqrstuvwxyz
zzzzzzzzzzzzRrnopqrstuvwxyz
zzzzzzzzzzzzTtnopqrstuvwxyz
zzzzzzzzzzzzUunopqrstuvwxyz
zzzzzzzzzzzzVvnopqrstuvwxyz
zzzzzzzzzzzzWwnopqrstuvwxyz
zzzzzzzzzzzzXxnopqrstuvwxyz
zzzzzzzzzzzzYynopqrstuvwxyz
zzzzzzzzzzzzZznopqrstuvwxyz
zzzzzzzzzzzzzSnopqrstuvwxyz
zzzzzzzzzzzzzNnopqrstuvwxyz
zzzzzzzzzzzzzOoopqrstuvwxyz
zzzzzzzzzzzzzPpopqrstuvwxyz
zzzzzzzzzzzzzQqopqrstuvwxyz
zzzzzzzzzzzzzRropqrstuvwxyz
zzzzzzzzzzzzzTtopqrstuvwxyz
zzzzzzzzzzzzzUuopqrstuvwxyz
zzzzzzzzzzzzzVvopqrstuvwxyz
zzzzzzzzzzzzzWwopqrstuvwxyz
zzzzzzzzzzzzzXxopqrstuvwxyz
zzzzzzzzzzzzzYyopqrstuvwxyz
zzzzzzzzzzzzzZzopqrstuvwxyz
zzzzzzzzzzzzzzSopqrstuvwxyz
zzzzzzzzzzzzzzOopqrstuvwxyz
zzzzzzzzzzzzzzPppqrstuvwxyz
zzzzzzzzzzzzzzQqpqrstuvwxyz
zzzzzzzzzzzzzzRrpqrstuvwxyz
zzzzzzzzzzzzzzTtpqrstuvwxyz
zzzzzzzzzzzzzzUupqrstuvwxyz
zzzzzzzzzzzzzzVvpqrstuvwxyz
zzzzzzzzzzzzzzWwpqrstuvwxyz
zzzzzzzzzzzzzzXxpqrstuvwxyz
zzzzzzzzzzzzzzYypqrstuvwxyz
zzzzzzzzzzzzzzZzpqrstuvwxyz
zzzzzzzzzzzzzzzSpqrstuvwxyz
zzzzzzzzzzzzzzzPpqrstuvwxyz
zzzzzzzzzzzzzzzQqqrstuvwxyz
zzzzzzzzzzzzzzzRrqrstuvwxyz
zzzzzzzzzzzzzzzTtqrstuvwxyz
zzzzzzzzzzzzzzzUuqrstuvwxyz
zzzzzzzzzzzzzzzVvqrstuvwxyz
zzzzzzzzzzzzzzzWwqrstuvwxyz
zzzzzzzzzzzzzzzXxqrstuvwxyz
zzzzzzzzzzzzzzzYyqrstuvwxyz
zzzzzzzzzzzzzzzZzqrstuvwxyz
zzzzzzzzzzzzzzzzSqrstuvwxyz
zzzzzzzzzzzzzzzzQqrstuvwxyz
zzzzzzzzzzzzzzzzRrrstuvwxyz
zzzzzzzzzzzzzzzzTtrstuvwxyz
zzzzzzzzzzzzzzzzUurstuvwxyz
zzzzzzzzzzzzzzzzVvrstuvwxyz
zzzzzzzzzzzzzzzzWwrstuvwxyz
zzzzzzzzzzzzzzzzXxrstuvwxyz
zzzzzzzzzzzzzzzzYyrstuvwxyz
zzzzzzzzzzzzzzzzZzrstuvwxyz
zzzzzzzzzzzzzzzzzSrstuvwxyz
zzzzzzzzzzzzzzzzzRrstuvwxyz
zzzzzzzzzzzzzzzzzTtstuvwxyz
zzzzzzzzzzzzzzzzzUustuvwxyz
zzzzzzzzzzzzzzzzzVvstuvwxyz
zzzzzzzzzzzzzzzzzWwstuvwxyz
zzzzzzzzzzzzzzzzzXxstuvwxyz
zzzzzzzzzzzzzzzzzYystuvwxyz
zzzzzzzzzzzzzzzzzZzstuvwxyz
zzzzzzzzzzzzzzzzzzSstuvwxyz
zzzzzzzzzzzzzzzzzzTttuvwxyz
zzzzzzzzzzzzzzzzzzUutuvwxyz
zzzzzzzzzzzzzzzzzzVvtuvwxyz
zzzzzzzzzzzzzzzzzzWwtuvwxyz
zzzzzzzzzzzzzzzzzzXxtuvwxyz
zzzzzzzzzzzzzzzzzzYytuvwxyz
zzzzzzzzzzzzzzzzzzZztuvwxyz
zzzzzzzzzzzzzzzzzzzStuvwxyz
zzzzzzzzzzzzzzzzzzzTtuvwxyz
zzzzzzzzzzzzzzzzzzzUuuvwxyz
zzzzzzzzzzzzzzzzzzzVvuvwxyz
zzzzzzzzzzzzzzzzzzzWwuvwxyz
zzzzzzzzzzzzzzzzzzzXxuvwxyz
zzzzzzzzzzzzzzzzzzzYyuvwxyz
zzzzzzzzzzzzzzzzzzzZzuvwxyz
zzzzzzzzzzzzzzzzzzzzSuvwxyz
zzzzzzzzzzzzzzzzzzzzUuvwxyz
zzzzzzzzzzzzzzzzzzzzVvvwxyz
zzzzzzzzzzzzzzzzzzzzWwvwxyz
zzzzzzzzzzzzzzzzzzzzXxvwxyz
zzzzzzzzzzzzzzzzzzzzYyvwxyz
zzzzzzzzzzzzzzzzzzzzZzvwxyz
zzzzzzzzzzzzzzzzzzzzzSvwxyz
zzzzzzzzzzzzzzzzzzzzzVvwxyz
zzzzzzzzzzzzzzzzzzzzzWwwxyz
zzzzzzzzzzzzzzzzzzzzzXxwxyz
zzzzzzzzzzzzzzzzzzzzzYywxyz
zzzzzzzzzzzzzzzzzzzzzZzwxyz
zzzzzzzzzzzzzzzzzzzzzzSwxyz
zzzzzzzzzzzzzzzzzzzzzzWwxyz
zzzzzzzzzzzzzzzzzzzzzzXxxyz
zzzzzzzzzzzzzzzzzzzzzzYyxyz
zzzzzzzzzzzzzzzzzzzzzzZzxyz
zzzzzzzzzzzzzzzzzzzzzzzSxyz
zzzzzzzzzzzzzzzzzzzzzzzXxyz
zzzzzzzzzzzzzzzzzzzzzzzYyyz
zzzzzzzzzzzzzzzzzzzzzzzZzyz
zzzzzzzzzzzzzzzzzzzzzzzzSyz
zzzzzzzzzzzzzzzzzzzzzzzzYyz
zzzzzzzzzzzzzzzzzzzzzzzzZzz
zzzzzzzzzzzzzzzzzzzzzzzzzSz
zzzzzzzzzzzzzzzzzzzzzzzzzZz
zzzzzzzzzzzzzzzzzzzzzzzzzzS
zzzzzzzzzzzzzzzzzzzzzzzzzz F
